test/enquirer.js

```diff
@@ -297,42 +287,44 @@ describe('Enquirer', function() {
 
     it('should register a custom prompt type as a class', () => {
       class Foo extends Input {}
-      enquirer.register('foo', Foo);
+
       enquirer = new Enquirer({
         show: false,
         autofill: true
       }, {
         color: 'orange'
-      })
+      });
+      enquirer.register('foo', Foo);
```

Comment:
This one's interesting, I had to move it down (below the Enquirer instantiation) because it seemed that having registration work on all instances of Enquirer was a side-effect rather than a feature. 

Please correct me if I'm wrong and I'll update the code to pass this test like it was originally.